### ISSUE TITLE
Issues with Elementary OS Loki toolbar buttons and dialogs

### ISSUE THREAD

**Issue by nova-ind** [LinxGem33/OSX-Arc-White#5]
Hi!

Great work on the theme, very well made. I have a few issues though, with the toolbar menu in some applications and dialogs such as the save file one in Chrome. I am on Elementary OS Loki.

Here is a screenshot of the toolbar issue (which happens in Sublime Text, GIMP, FileZilla, PoEdit, and quite possibly more apps that I haven't tested yet).
![screenshot from 2017-01-31 13 20 00](https://cloud.githubusercontent.com/assets/10091929/22463071/b01c191e-e7b8-11e6-90ca-d7a208cb4dc8.jpeg)
As you can see, when a button is focused it gets a white background and white text, and the same happens when I hover a menu item (my cursor was under "Settings").

And the dialog issue:
![screenshot from 2017-01-31 13 20 33](https://cloud.githubusercontent.com/assets/10091929/22463052/9ffd74c4-e7b8-11e6-91e9-a2f5b4c1bb5a.jpeg)
This has the ugly Win ~95 look.

Elementary is up-to-date, and the theme is the latest version as well (I just installed it yesterday).

I did not install gnome-shell as I read that it uses different versions of GTK than Elementary and I'm afraid of screwing things up and ending up with a non-usable installation.

Please let me know if that is the cause (and if I can safely install gnome-shell) or it is indeed a bug.

Thanks, and keep up the good work!

**Comment by LinxGem33** [comment]
@nova-ind 

appreciate the feed back!

Ill start a VM and look into the issue, can i ask did you download this from my official github page or my gnome look page both are up to date.

i ask because there are quite a few forks of my themes and i just want to make sure before i investigate, the theme is quite stable on elementary so its quite an unusual issue, i suspect it has more to do with elementary OS rather than the theme.

kind regards

**Comment by nova-ind** [comment]
Thanks for the prompt response.

I installed the theme from gnome-look, [here](https://www.gnome-look.org/p/1167049/).

The issue is not Sublime Text specific though. I have tested more apps, and have the same issues with toolbar menus, context menus, and dialog boxes for most applications installed on my system, which include: FileZilla, GIMP, Inkscape, GitEye, GParted, Virtualbox, Thunderbird, Sublime Text and PoEdit.

Applications which display properly: Glade interface designer, Grub customizer, Chrome, Pantheon Mail, System monitor, Photos and Synaptic package manager.

From what little experience I have, I gather the ones that work use a newer/different version of GTK? I have no idea how to find out which app uses what version of GTK though...

Let me know if I can help troubleshoot in any way!

Thanks

**Comment by LinxGem33** [comment]
@nova-ind 

I can now confirm that this does indeed affect multiple applications after further testing, I'm going to have to cross reference the elementary GTK stack and see if they have made any changes, i suspect that it most likely will be the case.

In the meantime can you test the other variants of the themes i.e OSX-Arc-Shadow & Darker and let me know if this issue still persists on your system.

kind regards

**Comment by nova-ind** [comment]
Darker has the same problem, that was my first choice of theme but I switched to White because the issue is made less apparent by the lower contrast.

Shadow makes the menus more usable, i.e. the text for active/hovered items is readable, but it has no hover state for items inside drop-down and context menus, and the borders are very thick and pronounced (screenshot attached, the cursor was over "Settings"). The toolbar buttons work fine though, including hover state.
![screenshot from 2017-02-01 11 38 58](https://cloud.githubusercontent.com/assets/10091929/22501847/3b38177c-e873-11e6-96c4-c4fdbee77608.png)

**Comment by nova-ind** [comment]
Hi,

Just wantedd to let you know, GitEye recently had an update to version 2.0.0.201701241830 which fixed a lot of the visual problems. Maybe a look through their changelog will help with troubleshooting.

Here's a screenshot of the new interface. The toolbar and context menus look perfect now, as well as the buttons and there are no more weird areas in the top bar.
![screenshot from 2017-02-15 12 47 48](https://cloud.githubusercontent.com/assets/10091929/22971233/b7eb1dd6-f37d-11e6-8e78-c75d40d6ce4a.png)

There's a new issue now, however, as dialog boxes appear nested inside one another, but I think it's on their end as I didn't encounter this problem in any other apps.
![screenshot from 2017-02-15 12 54 06](https://cloud.githubusercontent.com/assets/10091929/22971377/27faa614-f37e-11e6-8d81-d9cf5d64391d.png)

Cheers!

**Comment by LinxGem33** [comment]
@nova-ind 

appreciate the update, i will take a look at GitEye, I’m still unable to pin point the the issue so your insight is welcome news as I’ve been unable to make any significant progress addressing this issue.
I will leave this issue open a while just in case anybody else is having the same issue and hopefully ill have a fix soon or at least find out exactly what the issue is.

kind regards

**Comment by nova-ind** [comment]
Hey,

I was playing around with other themes and icon packs, and one of them required gtk2-engines-pixbuf. So I installed it, tried one more theme and eventually went back to this one, and BAM! Everything looks great, menus, dialogs, the works. Screenshot of the Chrome save file dialog:

![screenshot from 2017-02-21 17 30 13](https://cloud.githubusercontent.com/assets/10091929/23171628/ad780c46-f85b-11e6-985b-6a2edc2c25f1.png)

I don't know if it was pixbuf, an update to Elementary or another dependency of the themes I installed (VimixDark and Paper). I've also installed the Paper and Moka icon packs.

I hope this info will come in handy to help you close in on what the problem was and document it in case anybody else comes across it, because looking at the other themes and even reverting back to the Elementary one made me realize none came close to looking as good as this, even with the issues it had. Now that those issues are fixed, it is much better, and I'll definitely use it as my look for a long time.

So, thank you for the work you put in! And let me know if I can provide other info.

Cheers!

**Comment by LinxGem33** [comment]
@nova-ind 

really appreciate you testing the theme regarding the issue I’m not a 100% but i think you might be right about pixbuf because gtk2-engines-murrine uses (Debian, Ubuntu, elementary OS) so it seems logical that it might be a missing dependency anyway thank you again for taking the time and if you want to get more involved with the development of this or any other of my themes Id be glad to have you as a tester.

kind regards

**Comment by alxt17** [comment]
using Ubuntu 16.04 LTS offical, just ran into this problem, feeling sad for some ugly 95 style. 
luckily i read this post and get it fixed by installing gtk2-engines-pixbuf.
really appreciate nova-ind.

btw this is really a nice theme! love the arc-white-plus! nice work and thanks!

**Comment by LinxGem33** [comment]
@alxt17 

appreciate the feed back and conformation of the issue I’m more sure now its a missing dependency from the gtk2-engines, I’m marking this as solved but will leave it open a little while longer just in case other users have the same issue.

kind regards

**Comment by nova-ind** [comment]
Hi @LinxGem33,

It's the least I could do, I'm glad I was able to help you out a bit. I would like to get involved more, just let me know what I can do! I installed GTKInspector as well and am slowly learning more and more about how everything fits together, so someday soon I might even issue a pull request.

@alxt17 Glad I could help! And yes, it's really nice, I enjoy every minute I use this system :)

Cheers
